tests/locales.test.ts

```diff
@@ -2,15 +2,42 @@ import { Brazil_PtBr } from 'romcal/dist/bundles/brazil';
 import { France_Fr } from 'romcal/dist/bundles/france';
 
 import { Romcal } from '../lib';
+import { laOrdinalNumberGenderFixture } from './fixtures/la-ordinal-number-gender.fixture';
 
 describe('Testing localization functionality', () => {
-  test('If the locale is set to "fr", romcal should output text in French', async () => {
-    const date = await new Romcal({ localizedCalendar: France_Fr }).getOneLiturgicalDay('all_saints');
-    expect(date?.name).toBe('Tous les Saints');
+  describe('Test the French locale', () => {
+    test('If the locale is set to "fr", romcal should output text in French', async () => {
+      const date = await new Romcal({ localizedCalendar: France_Fr }).getOneLiturgicalDay('all_saints');
+      expect(date?.name).toBe('Tous les Saints');
+    });
   });
 
-  test('If the locale is set to "pt-BR", romcal should output text in Portuguese', async () => {
-    const date = await new Romcal({ localizedCalendar: Brazil_PtBr }).getOneLiturgicalDay('all_saints');
-    expect(date?.name).toBe('Todos os Santos');
+  describe('Test the Brazilian Portuguese locale', () => {
+    test('If the locale is set to "pt-BR", romcal should output text in Portuguese', async () => {
+      const date = await new Romcal({ localizedCalendar: Brazil_PtBr }).getOneLiturgicalDay('all_saints');
+      expect(date?.name).toBe('Todos os Santos');
+    });
+  });
+
+  describe('Test the Latin locale', () => {
+    describe('Testing whether Latin celebration names use correct gender for ordinal numbers', () => {
+      for (const i of laOrdinalNumberGenderFixture) {
+        const r = new Romcal({ localizedCalendar: i.c });
+
+        for (const d of i.t) {
+          test(`- \`${i.c.i18n.id}\`: \`${d.k}\``, async () => {
+            const day = await r.getOneLiturgicalDay(d.k)
+
+            if (day) {
+              /* eslint-disable-next-line jest/no-conditional-expect */
+              expect((day)?.name).toBe(d.v);
+            } else {
+              /* eslint-disable-next-line no-console */
+              console.log(`\`${i.c.i18n.id}\`: \`${d.k}\` is not celebrated in ${new Date().getUTCFullYear()}`)
+            }
+          });
+        }
+      }
```

Comment:
I'm realizing i don't think the single-character names of fixture fields is very readable.. it's probably too late to fix that, but if we could, that would be much appreciated. this is probably the downside of reviewing in a browser vs coding in an ide.

Reply:
I have no problem renaming the single-character names. It is not too late for this, as it is not yet merged into `dev` and it will never be too late, as this `type` is used in tests only, thus we can change them any time, even after merging it to `dev`.

How about using this `type` definition:

```ts
type Fixture = ({
  /** Base romcal bundle instance */
  calendar: BaseRomcalBundle,
  /** Localization tests definition */
  tests: ({
    /** Liturgical celebration localization key name */
    key: string,
    /** Expected value of a localized liturgical celebration name */
    value: string,
  })[]
})[]
```

Reply:
Look at [this](https://github.com/romcal/romcal/pull/477/files#diff-03515950f846611766a0962c19a6c41f320aeb8b07fbe8d90607d6f2869dfef7). I fixed it in #477 fixed, then I want to rebase this PR on top of it.